src/streaming/text/NotFragmentedTextBufferController.js

```diff
@@ -145,8 +145,6 @@ function NotFragmentedTextBufferController(config) {
     }
 
     function onDataUpdateCompleted(e) {
-        if (e.sender.getStreamId() !== streamInfo.id || e.sender.getType() !== type || e.error) return;
```

Comment:
This class is missing the getStreamId() function

Reply:
yes, I'm fixing it